oc_cdtapi/PgAPI.py

```
@@ -379,6 +379,26 @@ def get_task_custom_filter(self, **kwargs):
+        This method sends a GET request to get a client list including the ftp upload option by code.
+
+        Returns:
+            requests.Response: The response object from the API call.
+
+        Example:
+            >>> code = 2
+            >>> result = self.get_clients_list(code)
+            >>> print(result.status_code)
+            200
+        """
+        req = f"rest/api/1/clients"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)